api/db.mjs

```diff
@@ -0,0 +1,17 @@
+const get = async (key) => (await NAN.getWithMetadata(key))?.metadata
+const set = (key, metadata) => NAN.put(key, '', { metadata })
+const put = (key, value, opts) => NAN.put(key, value, opts)
+const update = async (key, p) => set(key, { ...(await get(key)), ...p })
+const find = async (prefix) => (await NAN.list({ prefix, limit: 1 })).keys[0]
+const filter = async (prefix) => {
+  let cursor, list
+  const results = []
+  while ((list = await NAN.list({ prefix, cursor }))) {
+    results.push(...list.keys)
+    if (list.list_complete) break
+    cursor = list.cursor
+  }
+  return results
+}
+
+export { get, set, put, update, find, filter }
```

Comment:
This is a ligth wrapper around [Cloudflare KV](https://developers.cloudflare.com/workers/runtime-apis/kv).
KV is the database we can use from cloudflare workers, it can store a lot of data, but for relatively small data, it can fit in the metadata, so this allow to do simply that.